### PR TITLE
example of azure pipelines for using PowerShell in an Windows Image

### PR DESCRIPTION
**Pull request by CTY-git**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,8 @@
-# example-azure-pipelines
-An example repo demonstrating how to integrate srcclr agent with Azure Pipelines
+# [:] Using SourceClear via Azure DevOps
```
Comment (spencerxiao) [inline README.md]: The formal name is Azure Pipelines instead of Azure DevOps

File: README.md [hunk 2]
```diff
@@ -1,2 +1,8 @@
-# example-azure-pipelines
-An example repo demonstrating how to integrate srcclr agent with Azure Pipelines
```
Comment (spencerxiao) [inline README.md:2]: append "running Windows PowerShell"

File: README.md (hunk 1 above)
Comment (CTY-git) [reply README.md]: changes should be reflected in commit 4e64bb3

File: README.md (hunk 2 above)
Comment (CTY-git) [reply README.md:2]: Appended on commit 6de6fb1

**Pull request merged**